### PR TITLE
[connector, exporter/datadog] Enable compute_stats_by_span_kind and peer_tags_aggregation by default

### PR DESCRIPTION
**Pull request by songy23**
#### Description
Flip configs `traces::compute_stats_by_span_kind`, `traces::peer_tags_aggregation` and `traces::peer_service_aggregation` to true by default

### CODE REVIEW

File: .chloggen/dd-conn-peer-tags.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadogconnector
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Flip configs `traces::compute_stats_by_span_kind`, `traces::peer_tags_aggregation` and `traces::peer_service_aggregation` to true by default"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: []
```
Comment (songy23) [inline .chloggen/dd-conn-peer-tags.yaml:13]:
```suggestion
issues: [35969]
```

File: .chloggen/dd-exp-peer-tags.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadogexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Flip configs `traces::compute_stats_by_span_kind`, `traces::peer_tags_aggregation` and `traces::peer_service_aggregation` to true by default"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: []
```
Comment (songy23) [inline .chloggen/dd-exp-peer-tags.yaml:13]:
```suggestion
issues: [35969]
```

**Review by mx-psi — APPROVED** [review]
LGTM. Do we need to request review from any other team at Datadog? Do we need to wait to merge this by some particualr date?

**Comment by rickbatka** [comment]
Hi @mx-psi and @songy23 , yes we need to wait to merge this, likely a few weeks. How do you want to handle that? Is there a tag to block merging? Or I could be added as a required reviewer and I'll withhold my approval until we are ready?

**Review by mx-psi — DISMISSED** [review]
Blocking merge per the comment above

**Review by pkalmakis — APPROVED** [review]

**Comment by rickbatka** [comment]
Hi @mx-psi and @songy23 , we are ready to merge this PR now!

**Review by mx-psi — APPROVED** [review]

**Pull request merged**